Resources/doc/index.rst

```diff
@@ -0,0 +1,45 @@
+Installation
+============
+
+Step 1: Download the Bundle
+---------------------------
+
+Open a command console, enter your project directory and execute the
+following command to download the latest stable version of this bundle:
+
+.. code-block:: bash
+
+    $ composer require symfony/monolog-bundle "~2.7"
```

Comment:
It isn't, as it means the 2.7 docs would make people install 3.0 (if that version is released) as it would just take the latest stable version if you don't specify a version. This could lead to much confusion.